lib/perl/Genome/Model/Tools/Analysis/Coverage/CoveragePlot.pm

```diff
@@ -0,0 +1,1439 @@
+package Genome::Model::Tools::Analysis::Coverage::CoveragePlot;
+
+use strict;
+use Genome;
+use IO::File;
+use warnings;
+use Carp;
+
+use File::Basename;
+use File::Spec::Functions;
+use POSIX qw(ceil floor log log10);
+
+use Statistics::R;
+
+
+class Genome::Model::Tools::Analysis::Coverage::CoveragePlot{
+    is => 'Command',
+    has => [
+        roi_file_path => {
+	    is => 'String',
+	    is_optional => 0,
+            doc => 'Region-of-interest file in the BED format',
+        },
+	
+	output_directory => {
+	    is => 'String',
+	    is_optional => 0,
+	    doc => 'Directory to save output files such as PDF file and tables for plotting, and ref-cov stats files',
+	},
+	
+	bam_files => {
+	    is => 'String',
+	    is_optional => 1,
+	    doc => 'Comma-separated list of bam files to generate coverage plots from, Must specify either --bam_files or --refalign_models',
+	},
```

Comment:
If you specify this (and other comma-separated properties) as `is_many => 1` you don't have to do any splitting up into an array during execution. UR will do that for you when you call `$self->bam_files`.